cmd/notation/policy/import.go

```diff
@@ -0,0 +1,106 @@
+package policy
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/spf13/cobra"
+)
+
+type importOpts struct {
+	cmd.LoggingFlagOpts
+	filePath string
+	force    bool
+}
+
+func importCmd() *cobra.Command {
+	var opts importOpts
+	command := &cobra.Command{
+		Use:   "import [flags] <file_path>",
+		Short: "Import trust policy configuration from a JSON file",
+		Long: `Import trust policy configuration from a JSON file.
+Example - Import trust policy configuration from a file:
+  notation policy import my_policy.json
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.filePath = args[0]
+			return runImport(cmd, opts)
+		},
+	}
+	command.Flags().BoolVar(&opts.force, "force", false, "override the existing trust policy configuration, never prompt")
+	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
+	return command
+}
+
+func runImport(command *cobra.Command, opts importOpts) error {
+	// optional confirmation
+	policyPath := filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy)
+	if _, err := os.Stat(policyPath); err == nil {
+		confirmed, err := opts.askForConfirmation(os.Stdin, "Existed trust policy configuration found, do you want to overwrite it?")
+		if err != nil {
+			return err
+		}
+		if !confirmed {
+			return nil
+		}
+	}
+
+	// read configuration
+	policyJson, err := os.ReadFile(opts.filePath)
+	if err != nil {
+		// todo: a tip to import trust policy configuration from a JSON file.
+		return fmt.Errorf("failed to read trust policy configuration: %w", err)
+	}
+
+	// parse and validate
+	var doc trustpolicy.Document
+	if err = json.Unmarshal(policyJson, &doc); err != nil {
+		return fmt.Errorf("failed to parse trust policy configuration: %w", err)
+	}
+	if err = doc.Validate(); err != nil {
+		return fmt.Errorf("failed to validate trust policy configuration: %w", err)
+	}
+
+	// write
+	if err = os.WriteFile(policyPath, policyJson, 0666); err != nil {
```

Comment:
What's the expected permission? `0600` or `0644`?

Reply:
Let's do it this way:
1) if the original policy file exists, clone the file mode from it
2) otherwise set it to `0644`

Reply:
Will change to minimal user access (https://github.com/notaryproject/notation/pull/453/files) as is suggested by @patrickzheng200